server/frontend.go

```diff
@@ -74,3 +105,93 @@ func (fe *Frontend) GetUpdates(args *common.GetUpdatesArgs, reply *common.GetUpd
 	// @TODO
 	return nil
 }
+
+// periodicWrite runs until the dead flag is set, and periodically send a write
+// request to all replicas telling them to advance their write epoch.
+func (fe *Frontend) periodicWrite() {
+	for fe.dead == 0 {
+		tick := time.After(fe.WriteInterval)
+		select {
+		case <-tick:
+			args := &common.ReplicaWriteArgs{
```

Comment:
This needs to be properly ordered among all writes. So we need to either assign it a global sequence number of piggyback this trigger to the next write.